presto-spi/src/main/java/com/facebook/presto/spi/security/GrantInfo.java

```diff
@@ -22,12 +22,12 @@
 public class GrantInfo
 {
     private final Set<PrivilegeInfo> privilegeInfo;
-    private final Identity grantee;
+    private final ConnectorIdentity grantee;
```

Comment:
This class is used both in context of connector metadata and global (`com.facebook.presto.metadata.Metadata`). For global use it should have `Identity` (not `ConnectorIdentity`). Should we have `ConnectorGrantInfo`?

Edit: Oh - Is see that `grantee` and `grantor` become `PrestoPrincipal` later on. Maybe put some comment here so reviewer is not confused.

Edit2: Or maybe make those two fields `PrestoPrincipal` earlier in commit flow.